content/learn/book/migration-guides/0.6-0.7/_index.md

```diff
@@ -222,17 +222,15 @@ fn main() {
 struct Config(u32);
 
 fn local_config(local: u32) -> impl FnMut(ResMut<Config>) {
-    move |mut val| {
+    move |mut val: ResMut<Config>| {
```

Comment:
```suggestion
    move || {
```

As a note, you shouldn't need to specify this type, since it was already specified in the return type.

Reply:
Yep, but I saw a video of someone reviewing the migration guide and they were confused about what that parameter was. That's the reason I added it, not because it's necessary.

Reply:
Hmm. It's still idiomatic not to duplicate writing it out.

Reply:
Agreed, maybe I could add a comment. Something like

```suggestion
	// This closure will be the system that will be executed
    move || {
```